src/deadline/unreal_submitter/submitter.py

```diff
@@ -202,9 +244,9 @@ def submit_jobs(self) -> list[str]:
             self.submit_message = "Start submitting..."
             self._submission_failed_message = ""
 
-            t = threading.Thread(
-                target=self._start_submit, args=(job.job_bundle_path,), daemon=True
-            )
+            job_bundle_path = job.create_job_bundle()
```

Comment:
we just move bundle creation here to not spam with job bundles before submission itself, because there can be some errors on openjd model validations etc.